packages/shared/styles/components/atoms/SfRating.scss

```diff
@@ -3,8 +3,15 @@
   display: flex;
   &__icon {
     --icon-color: var(--c-primary);
+    --icon-color-negative: var(--c-gray-variant);
```

Comment:
@AdamPawlinski when you add coverage or fraction property you can add --icon-fraction in SfIcon scss and after that refactor this changes to adjust to these changes :)

Reply:
fixes made